Audible-Altimeter-Embedded/src/Tiny2350.cpp

```diff
@@ -0,0 +1,56 @@
+#include <Audible-Altimeter-Embedded/Tiny2350.hpp>
+#include <cstdint>
+
+#include "pico/stdlib.h"
+
+namespace {
+
+constexpr std::uint32_t LED_PIN_RED{18};
+constexpr std::uint32_t LED_PIN_GREEN{19};
+constexpr std::uint32_t LED_PIN_BLUE{20};
```

Comment:
lets move gpio pin assignment declarations to a central location? Maybe like a `board_description.hpp` or `hardware_declarations.hpp` that way we can see all of the board assets in one place. what do you think? For the audio pr I will need to declare a dma channel, pio block and some gpio pins.

Reply:
Yes, that sounds good like a good idea